Library/Formula/open-mesh.rb

```diff
@@ -16,16 +16,70 @@ class OpenMesh < Formula
   head "http://openmesh.org/svnrepo/OpenMesh/trunk/", :using => :svn
 
   depends_on "cmake" => :build
-  depends_on "qt"
+  depends_on "qt" => :optional
+
+  # For 3.3 version, when BUILD_APPS=OFF, the fixbundle.cmake will not be
+  # generated for a lacking cmake command `configure_file`. I patched the
+  # cmakelists.txt a little by adding the missing line.
+  patch do
+    url "https://gist.githubusercontent.com/autosquid/41741a5ff42ddb4d104c/raw/6c2b49b3b23bdbafbed053dc8dc54d3407cdfba1/CMakeLists.patch"
+    sha256 "cd7f8e58dce947f1680fc0cb10bacf208fc52b384025d1598eef31b9c6840528"
+  end
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!